modules/govuk_mysql/templates/usr/local/bin/xtrabackup_s3_restore.erb

```diff
@@ -40,17 +43,18 @@ else
   service mysql stop
 
   # Ensure that it's definitely not running before we move data directories
-  if /bin/pidof mysql
+  if /bin/pidof mysqld; then
     echo "MySQL still running, exiting"
     exit 1
   fi
 
-  # Move the current directory to a backup location
+  # Copy the current directory to a backup location; we cannot move this if
+  # the directory is a mount point so we need to copy
   echo "Backing up current dir"
-  mv /var/lib/mysql /var/lib/mysql-bak
+  cp -R /var/lib/mysql /var/lib/mysql-bak
```

Comment:
Hmm, true about the directory name, I can make that unique. With regards to backing up, I just want to ensure that if the script fails for whatever reason then we're not left with no database whatsoever.